libs/libcommon/include/common/iostream_debug_helpers.h

```diff
@@ -85,5 +85,29 @@ std::ostream & operator<<(std::ostream & stream, const std::pair<K, V> & what)
 	return stream;
 }
 
+#include <ratio>
+
+template < std::intmax_t Num,  std::intmax_t Denom>
```

Comment:
Inconsistent whitespace. Double whitespace.
Num with capital letter is not in style, lowercase will be Ok..